tester/src/test/java/com/silanis/esl/sdk/examples/AccountLimitSupportedLanguagesSettingsExampleTest.java

```diff
@@ -0,0 +1,47 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.api.model.SupportedLanguages;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class AccountLimitSupportedLanguagesSettingsExampleTest {
+
+    private SupportedLanguages supportedLanguages1 = getSupportedLanguages(Arrays.asList("en", "fr"), "fr");
+    private SupportedLanguages supportedLanguages2 = getSupportedLanguages(Arrays.asList("en", "de"), "de");
+    @Test
+    public void verifyResult() {
+        AccountLimitSupportedLanguagesSettingsExample example = new AccountLimitSupportedLanguagesSettingsExample();
+        example.run();
+
```

Comment:
Could you please check the resource is null before saving any resource value and after deleting the resource value ?